js/stat.js

```diff
@@ -0,0 +1,46 @@
+window.renderStatistics = function (ctx, names, times) {
+    ctx.fillStyle = 'rgba(0, 0, 0, 0.7)';
+    ctx.fillRect(110, 20, 420, 270);
+    ctx.fillStyle = 'rgba(256, 256, 256, 1)';
```

Comment:
```suggestion
    ctx.fillStyle = 'rgba(255, 255, 255, 1)';
```
цвета идут от 0 до 255